app/services/dnd_service.rb

```diff
@@ -63,8 +72,13 @@ def query_monster_details(index)
             name
             desc
           }
+          condition_immunities {
+            name
+          }
         }
       }
     GRAPHQL
+
+    response = dnd_client.execute(query, index: index)
```

Comment:
```suggestion
    dnd_client.execute(query, index: index)
```

We don't need to assign this to a variable as the variable isn't being used anywhere.

Reply:
All unnecessary variable declarations have now been removed from this service.